packages/loot-core/src/shared/util.ts

```diff
@@ -237,13 +237,14 @@ export function getNumberFormat({
   format?: NumberFormats;
   hideFraction: boolean;
 } = numberFormatConfig) {
-  let locale, regex, separator;
+  let locale, regex, separator, regexSeparator;
```

Comment:
Nit: 
```suggestion
  let locale, regex, separator, separatorRegex;
```

Reply:
@joel-jeremy Done, I renamed the variable.